doc/tools/coqrst/coqdomain.py

```diff
@@ -905,9 +905,17 @@ def is_coqtop_block(node):
         return isinstance(node, nodes.Element) and 'coqtop_options' in node
 
     @staticmethod
-    def split_sentences(source):
+    def split_sentences(node):
         """Split Coq sentences in source. Could be improved."""
```

Comment:
What the code does is clear to me since this is basically what I suggested on Gitter. However, I suggest refining the comment to be a bit more precise on the splitting strategy.

Reply:
Especially since the code doesn't split all sentences anymore, right?

Reply:
Right, we considered that if a documentation writer decided to put several sentences on the same line this is on purpose and probably means that the intermediate state is less important.

Reply:
Sounds like a very good idea.